### ISSUE TITLE
Linter doesn't seem to respect jshintrc file

### ISSUE THREAD

**Issue by lombardi-gux** [AtomLinter/linter-jshint#38]
I installed the Linter plugin and tried 2 different jshint plugins. I'm confused because in both cases the error are listed as "jslint", but the bigger issue is that it flags a lot of errors that are not really errors.

For example, I have a Grunt-based project. I am using jshint as a Grunt tasks as well. When I first started using it, I got all sorts of "errors" about white space or "use the function form of use strict" etc. I looked up those errors and created a jshintrc file in the project root. Now the Grunt task does not flag any of these errors, but the Atom Linter does.

**Comment by klebba** [comment]
+1 for the restart tip. This should be noted at the top of the README

**Comment by irrationalistic** [comment]
I'm having a similar issue, but it more specifically refers to the plugin seeming to ignore my global .jshintrc found at the user root directory `~`. Is there any way to retain that recursive searching up until a jshintrc is found or it hits `/` ? Restarting the editor doesn't allow it to pick up the file change.

As an FYI, my .jshintrc file at `~`:
```
{
  "node": true
}
```

**Comment by bucaran** [comment]
@klebba +1

Just experienced this myself. Reloading (View->Reload) Atom also works.

**Comment by crucialfelix** [comment]
I found that if there are any errors in the .jshintrc then you only find out about them by looking at a js file and examining the error flag on the very first character. 

eg. jasmine: true is no longer valid and you now have to do predef: ['expect', 'it', 'describe']

**Comment by Eschon** [comment]
Just had this problem. I have a very simple .jshintrc file:
```JavaScript
{
  "esnext": true
}
```
I tried reloading and restarting Atom but linter-jshint keeps complaining about arrow functions. The strange thing is I have another project where i have the same .jshintrc file except that I forgot to surround the `esnext` with quotes and that works. Even though the command line tool can't parse that config file.

**Comment by steelbrain** [comment]
Fixed now.
